scripts/run-integration-tests.sh

```diff
@@ -270,6 +248,35 @@ if [[ $TEST_PASS -eq 0 ]]; then
   emit_cloudwatch_metric "integration_test_status" "1"
 fi
 
+if [[ $RUN_CALICO_TEST == true ]]; then
+  echo "Starting Helm installing Tigera operator and running Calico STAR tests"
+  pushd ./test
+  VPC_ID=$(eksctl get cluster $CLUSTER_NAME -oyaml | grep vpc | cut -d ":" -f 2 | awk '{$1=$1};1')
+  # we can automatically use latest version in Calico repo, or use the known highest version (currently v3.22.0)
+  calico_version=$CALICO_VERSION
+  if [[ $RUN_LATEST_CALICO_VERSION == true ]]; then
+    version_tag=$(curl -i https://api.github.com/repos/projectcalico/calico/releases/latest | grep "tag_name") || true
+    if [[ -n $version_tag ]]; then
+      calico_version=$(echo $version_tag | cut -d ":" -f 2 | cut -d '"' -f 2 )
+    else
+      echo "Getting Calico latest version failed, will fall back to default/set version $calico_version instead"
+    fi
+  fi
+  echo "Using Calico version $calico_version to test"
+  ginkgo -v e2e/calico -- --cluster-kubeconfig=$KUBECONFIG --cluster-name=$CLUSTER_NAME --aws-region=$AWS_DEFAULT_REGION --aws-vpc-id=$VPC_ID --calico-version=$calico_version
+  popd
+  
+  if [[ "$DEPROVISION" == false ]]; then
```

Comment:
I think DEPROVISION is set to false only when we want to login to the cluster nodes and debug. As such, automated termination of nodes may not be needed here. Thoughts ?

Reply:
Are we planning to recycle the cluster? @jayanthvn

Reply:
I feel we should keep this in case the Calico tests interfere with any tests without awareness in the future.